articles/architecture-scenarios/b2c/branding.md

```diff
@@ -0,0 +1,122 @@
+
+Some of the items that you can brand include:
+
+* Login, Change Password, and Error pages
+* Custom domains
+* Email templates
+
+Auth0 also supports [internationalization (I18N)](/i18n) and localization (L10N), both of which are important if you work with an international clientele.
+
+## Design considerations
+
+For some users, the out-of-box experience -- perhaps with just a touch of alteration -- is more than sufficient. However, for those wanting more control over appearance and user experience, Auth0 provides extensive flexibility when it comes to customizing and configuring things like your [Login](#universal-login), [Change Password](#change-password), and [Error](#errors) pages, [Custom Domain](#custom-domain) and [Email Templates](#email-templates). This allows you to create the user experience look and feel you need.
+
+When considering the items you want to brand, as well as how best to brand these items, you'll want to consider the following.
+
+* Will you brand your login page?
+
+* Will you need to localize your login page?
+
+* Will you customize emails so that they're not just branded, but vary based on user preference?
+
+* How will users know that they're still on your domain when they see your login page?
+
+* What do you need to do to provide additional browser security (e.g., implement Extended Validation)?
+
+* How will you direct users in the event of errors?
+
+### Universal Login and Login Pages
+
+The [Universal Login](/universal-login) process is the recommended method for authenticating users, and it centers around the Login Page. You can customize the Login Page via the Dashboard to support your organization's branding requirements. You can make [simple changes](/universal-login#simple-customization) that include changes to the logo, primary color, or background color, or you can make more [advanced-level changes](/universal-login#advanced-customization) that require you to modify the script powering the page itself directly.
+
+If you opt to customize the Login Page script, we strongly recommend that you:
+
+* [Use version control](/universal-login/version-control) to maintain prior versions
+* Deploy changes to your script via [automated deployments](#)
+
+Once modified, you are responsible for maintaining the script powering your Login Page, including updating the version number of the Lock widget used.
+
+::: warning
+Exercise caution regarding the use of third-party JavaScript on your Login Page, since sensitive security-related information often flows through the page.
+:::
+
+The Login Page comes default with [multi-language support](/libraries/lock/v11/i18n). In our experience, almost all applications need internationalization/localization, and while it is fairly simple to add during the development phases, it can become difficult at a later stage. As such, we recommend, at the very least, accounting for this need if you do not implement either feature during this phase.
```

Comment:
```suggestion
The Login Page comes with [multi-language support](/libraries/lock/v11/i18n) provided by. In our experience, almost all applications need internationalization/localization, and while it is fairly simple to add during the development phases, it can become difficult at a later stage. As such, we recommend, at the very least, accounting for this need if you do not implement either feature during this phase.
```

Reply:
What is this change meant to do? The original is clearer in meaning to me?

Reply:
Helps if I remember to add all of the change! LOL! Updated it for your review :-)

Reply:
Haha that makes more sense, I probably could've pieced that together sorry 😀